src/ui/services/user.js

```diff
@@ -24,46 +18,69 @@ const getUser = async (
   console.log(url);
 
   await axios(url, config)
+    .then((response) => {
+      const data = response.data;
+      if (setData) {
+        setData(data);
+      }
+      if (setIsLoading) {
+        setIsLoading(false);
+      }
+    })
+    .catch((error) => {
+      if (error.response && error.response.status === 401) {
+        if (setAuth) {
+          setAuth(false);
+        }
+      } else {
+        if (setIsError) {
+          setIsError(true);
+        }
+      }
+      if (setIsLoading) {
+        setIsLoading(false);
+      }
+    });
+};
+
+const getUsers = async (setIsLoading, setData, setAuth, setIsError, query = {}) => {
+  const url = new URL(`${baseUrl}/api/v1/user`);
+  url.search = new URLSearchParams(query);
+  setIsLoading(true);
+  await axios(url.toString(), { withCredentials: true })
     .then((response) => {
       const data = response.data;
       setData(data);
-      console.log(data);
       setIsLoading(false);
     })
     .catch((error) => {
-      if (error.response && error.response.status === 401) setAuth(false);
-      else setIsError(true);
+      setIsLoading(false);
+      if (error.response && error.response.status === 401) {
+        setAuth(false);
+      } else {
+        setIsError(true);
+      }
       setIsLoading(false);
     });
 };
 
-const createUser = async (data) => {
+const updateUser = async (data) => {
   console.log(data);
-  const url = new URL(`${baseUrl}/auth/profile`);
-  await await axios
-    .post(url, data, { withCredentials: true })
-    .then(() => {})
-    .catch((error) => {
-      console.log(error.response.data.message);
-      throw error;
-    });
+  const url = new URL(`${baseUrl}/api/auth/gitAccount`);
+  await axios.post(url, data, { withCredentials: true }).catch((error) => {
+    console.log(error.response.data.message);
+    throw error;
+  });
```

Comment:
@JamieSlome I believe these changes are causing the test failures.

Reply:
@coopernetes - thanks for spotting 👍 I'll `.skip()` the failing test for now and we can re-enable once we introduce the manual creation of new user records via the API.